diagnostic.md

```diff
@@ -49,10 +56,22 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    I am not entirely sure what you mean by 'value'.
+    here is how you would get to that route in the router
+
```

Comment:
If we go to `movies/123` and we want to access that `123` in the Route object, we use `params.movie_id`.

0/1